### ISSUE TITLE
✅ ♿ 🐞 Bug Report: Accessibility issue button widget: role="button" attribute should be optional - [ED-10602]

### ISSUE THREAD

**Issue by Bovelett** [elementor/elementor#14908]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

The button widget generates this:
`<a href="#" class="elementor-button-link elementor-button elementor-size-sm" role="button">`
The role="button" prevents it from being listed in a link list in screen readers. A fact that many users, who do their best to create accessible content by using descriptive links, are blissfully unaware of.

This is a problem when the button widget is used as a CTA, for example. I sure want visitors depending on a screen reader to be able to hear the link in the link list.

### Currently, the only way to work around this issue is by creating a button with custom HTML. Which basically is copying the source code of the button widget, without the role="button".  Something the average Elementor users doesn't know how to do because you market as a "no code" solution.

The button widget, in this case, is nothing but a link fancily styled to look like a button. The role= "button" should ONLY apply if the button DOES something. When it triggers an action. Like a pop-up, or send a form.

UPDATE: 3 Dec. 2022: I removed the remark about the loops, it was unrelated. But the rest still stands.

It's a catch22. But for the sake of accessibility, I hope you can find a solution to this? 

Thank you,

Anne-Mieke

## Steps to reproduce

Insert a button with the button widget and inspect the code. You will find this:
`<a href="#" class="elementor-button-link elementor-button elementor-size-sm" role="button">
						<span class="elementor-button-content-wrapper">
						<span class="elementor-button-text">Click here</span>
		</span>
					</a>`

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

**Comment by IkeTen** [comment]
+1 to this @Bovelett

**Comment by isaacadu** [comment]
+1 to this @Bovelett

**Comment by zoobli-io** [comment]
+1 to this @Bovelett

**Comment by susanhop** [comment]
+1

**Comment by zeinnicholas** [comment]
+1

**Comment by VerdiH** [comment]
+1 for anything that improves accessibility. Thank you for sharing and caring @Bovelett 🙏

**Comment by nicholaszein** [comment]
Hey @Bovelett! 🤗 

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor v3.14.0! 🥳 

When a link is added to a button widget, the `role="button"` is removed automatically, which allows the link to be listed in a links list on screen reader software! Thank you for raising this issue and for being so patient!

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂

PS: Hey buddy @zeinnicholas! 🙃
